tekton-proposal.md

```diff
@@ -0,0 +1,73 @@
+# Knative Build and Tekton Pipelines Steering Committee proposal
+
+Jason Hall ([jasonhall@google.com](mailto:jasonhall@google.com))
+
+Feb 25 2019
+
+## Objective
+
+Propose a resolution to the ongoing confusion and FUD around the future of Knative Build with regard to the recently announced Tekton Pipelines change. This doc should be used as a forum for external comment and questions, to resolve any concerns before the Steering Committee decides whether to adopt the proposal.
```

Comment:
I would remove "FUD", confusion is strong enough.

Reply:
Can you add a reference to the tekton "announcement" ? or is that just a slack note or a TOC discussion ? Would be nice to point to a record.

Reply:
The closest I have to a link to the announcement is the 2/13 Build WG meeting where it was announced.

I think that's kind of the main problem with "the announcement", there was never a written public record of it, only WG meeting video, and it's not feasible to expect everyone to attend the meeting or watch the recording every week.